ouroboros-network/src/Ouroboros/Network/NodeToNode/Version.hs

```diff
@@ -33,16 +33,22 @@ data NodeToNodeVersion
     -- ^ Changes:
     --
     -- * Enable full duplex connections.
+    | NodeToNodeV_10
+    -- ^ Changes:
+    --
+    -- * Enable @CardanoNodeToNodeVersion6@, i.e., Babbage
```

Comment:
There is also `network-mux/tools/cardano-ping.hs` to update -- it has its own version data type, but the idea is the same. Should be mechanical to update

Reply:
Interesting. Updated!